systers_portal/meetup/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.conf.urls import url
+
+from meetup.views import MeetupLocationDetailView
+
+
+urlpatterns = [
+    url(r'^(?P<slug>\w+)/(?P<page_slug>\w+)/$',
```

Comment:
I can create a separate url for each menu item and there in url itself we can specify corresponding template name and model, we actually don't need views in this case.

For Eg: for about --> url(r'^(?P<slug>\w+)/about/$', DetailView.as_view(
        template_name='meetup/about.html', model=MeetupLocation), 
        name='about_meetup_location')

But I am not sure if it is good or not to not keep views. Need your views for this.